app/src/pages/inside/testCaseLibraryPage/testCaseList/testCaseSidePanel/testCaseSidePanel.scss

```diff
@@ -0,0 +1,207 @@
+/*
+ * Copyright 2025 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.test-case-side-panel {
```

Comment:
Can you please clarify why in some file you have the nesting of CSS styles (e.g. `app/src/components/collapsibleSection/collapsibleSection.scss`) and in some files like this you don't have it?

Reply:
made a nested structure